README.md

```diff
@@ -1,27 +1,24 @@
 # JIRA CLI [![Gem Version](https://badge.fury.io/rb/jira-cli.png)](http://badge.fury.io/rb/jira-cli)
 
-Ruby gem CLI tool used to manage JIRA workflows leveraging git
+Control JIRA from your terminal
 
 * * *
 
-[![Circle CI](https://circleci.com/gh/drn/jira-cli.svg?style=svg)](https://circleci.com/gh/drn/jira-cli)
-[![Dependency Status](https://gemnasium.com/darrenli/jira-cli.png)](https://gemnasium.com/darrenli/jira-cli)
-[![Code Climate](https://codeclimate.com/github/darrenli/jira-cli.png)](https://codeclimate.com/github/darrenli/jira-cli)
+[![Circle CI](https://circleci.com/gh/ruby-jira/jira-cli.svg?style=svg)](https://circleci.com/gh/ruby-jira/jira-cli)
+[![Code Climate](https://codeclimate.com/github/ruby-jira/jira-cli.png)](https://codeclimate.com/github/ruby-jira/jira-cli)
 
 ### Available Commands
 
-    jira all                # Describes all local branches that match JIRA ticketing syntax
     jira assign             # Assign a ticket to a user
```

Comment:
Several of these should now also be specifying `<ticket>` at a required parameter too, actually.